migration_toolkit_for_containers/mtc-migrating-vms.adoc

```diff
@@ -0,0 +1,79 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="mtc-migrating-vms"]
+= Migrating virtual machine storage
+include::_attributes/common-attributes.adoc[]
+:context: mtc-migrating-vms
+
+toc::[]
+
+{mtc-full} ({mtc-short}) 1.8.4 supports storage migration for KubeVirt virtual machines (VMs).
+
+The migration can occur offline, when the VM is turned off, or online, when the VM is running.
+
+The main use case for this storage migration for KubeVirt VMs is migration from one storage class to another. For example, if you are rebalancing between different storage providers, or if there is new storage available that is better suited to the workload running inside the VM.
+
+== Supported persistent volume actions
+
+[width="100%",cols="31%,33%,36%",options="header",]
+|===
+|Action
+|Supported
+|Description
+
+|Copy
+|Yes
+a|
+. Create a new persistent volume (PV) in the same namespace.
+. Copy data from the source PV to the target PV, and modify the VM definition to point to the new PV.
+** If the `liveMigrate` flag is set, the VM migrates live.
+** If the `liveMigrate` flag is not set, the VM shuts down, and the source PV contents are copied to the target PV and the VM is restarted.
+
+|Move
+|No
+|This action is not supported.
+|===
+
+== Preparing for migration
+
+To prepare for migration, deploy mig-controller and Velero, and then create the following `Mig`' resources on the cluster where the migration controller is running to prepare for migration:
+
+.Procedure
+
+. Deploy link:https://github.com/kubevirt/kubevirt/[KubeVirt]. In order to support storage live migration, you need to deploy KubeVirt version 1.3.0 or later. Earlier versions do not support storage live migration. You also need to configure KubeVirt to enable storage live migration according to the link:https://kubevirt.io/user-guide/storage/volume_migration/[Volume Migration].
+
+. Deploy the link:https://github.com/kubevirt/containerized-data-importer/?tab=readme-ov-file#deploy-it[Containerized Data Importer (CDI)]
```

Comment:
Don't need to mention this here. CDI and KubeVirt are part of OpenShift Virtualization. So maybe point to the installation instructions for OCP-V here. https://docs.openshift.com/container-platform/4.16/virt/install/installing-virt.html

Reply:
> So maybe point to the installation instructions for OCP-V here. https://docs.openshift.com/container-platform/4.16/virt/install/installing-virt.html

@awels , is the use of version-specific (`4.16`) preferred over `latest` which will redirect to the latest available released version -- currently, 4.16?  https://docs.openshift.com/container-platform/latest/virt/install/installing-virt.html

Reply:
No, we should use latest there. I just pointed to one I could find easily.